app/views/components/_contents-list.html.erb

```diff
@@ -16,20 +16,47 @@
       <%= t("content_item.contents") %>
     </h2>
     <ol class="app-c-contents-list__list">
-      <% contents.each do |contents_item| %>
+      <% contents.each.with_index(1) do |contents_item, position| %>
+        <%
+          active_class = contents_item[:active]? "app-c-contents-list__list-item--active" : ""
+          aria_current = contents_item[:active]? "aria-current = true" : ""
+        %>
         <li
-          class="app-c-contents-list__list-item app-c-contents-list__list-item--<%= parent_list_item_modifier %>"
-        >
-          <%= link_to contents_item[:href], class: 'app-c-contents-list__link' do %>
-            <%= format_numbers ? wrap_numbers_with_spans(contents_item[:text]) : contents_item[:text] %>
-          <% end %>
+           class = "app-c-contents-list__list-item app-c-contents-list__list-item--<%= parent_list_item_modifier %> <%= active_class %>"
+           <%= aria_current %> >
```

Comment:
I'm not sure the aria-current attribute is being read out where it is currently positioned on the `li` element. When I tested, Voiceover didn't read anything when the element was 'current'. It may need to be moved to the text itself?

Reply:
I've tested here and with a scratch html file locally but I can't get it to work either - I've double checked the implementation, but it looks like support is coming in slowly - according to the table on this page http://design-patterns.tink.uk/aria-current/ it should work in the latest MacOS so perhaps that's the problem.